src/styles/product-card.css

```
@@ -0,0 +1,90 @@
+.product-card {
+  width: min-content;
+  min-width: 200px;
```

Щирина картки не повинна змінюватись в залежності від контенту.
```suggestion
  width: 200px;
```